search-sentiments-map/public/scripts/map-script.js

```diff
@@ -86,24 +101,25 @@ function updateLegends() {
       DATA_MAX.toLocaleString();
 }
 
-/** Loads the country boundary polygons from a GeoJSON source. */
-function loadMapOutline() {
-  map.data.loadGeoJson('countries.geojson', null);
-}
-
 /** 
  * Gets the selected mode (sentiment or popularity) from the webpage and loads
- * corresponding data.
+ * corresponding data on the map.
  */
-function loadCountryDataByMode() {
+function loadRegionDataByMode() {
   isSentimentMode = !document.getElementById('sentiment-popularity-check').checked;
 
   const topicHeader = document.getElementById('topic-header');
-  topicHeader.innerText = isSentimentMode ?
-      'Worldwide sentiment scores of search results for "' + getCurrentSearchData().topic + '"' :
-      'Worldwide search interest scores for "' + getCurrentSearchData().topic + '"' ;
+  if (isWorldLevel) {
+    topicHeader.innerText = isSentimentMode ?
+        'Worldwide sentiment scores of search results for "' + getCurrentSearchData().topic + '"' :
+        'Worldwide search interest scores for "' + getCurrentSearchData().topic + '"';
+  } else {
+    topicHeader.innerText = 
+        'State-level search interest scores for "' + getCurrentSearchData().topic + '"';
+  }
```

Comment:
Could we make this one big conditional?

Reply:
seriously?

Reply:
I mean both times you are adding to topicHeader.innerText

Reply:
But the first if block cannot be shortened to one line

Reply:
Okay then.